cmd/nvidia-cdi-hook/cudacompat/cudacompat.go

```diff
@@ -103,53 +102,57 @@ func (m command) run(_ *cli.Context, cfg *options) error {
 		return fmt.Errorf("failed to load container state: %w", err)
 	}
 
-	containerRootDir, err := s.GetContainerRoot()
+	containerRootDirPath, err := s.GetContainerRootDirPath()
 	if err != nil {
 		return fmt.Errorf("failed to determined container root: %w", err)
 	}
 
-	containerForwardCompatDir, err := m.getContainerForwardCompatDir(containerRoot(containerRootDir), cfg.hostDriverVersion)
+	containerForwardCompatDir, err := m.getContainerForwardCompatDirPathInContainer(containerRootDirPath, cfg.hostDriverVersion)
```

Comment:
Thanks for having added the `InContainer` suffix here. I know these things get lengthy but the information density is really good and this makes stuff no-brainer readable, which was the goal.

And again, the `containerRootDirPath` can be thought of as a `containerRootDirPathOnHost`, right?

Reply:
Yes, the variable should be `containerRootDirPathOnHost`.